launchable/commands/verify.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import click
+import platform
+
+from ..utils.env_keys import REPORT_ERROR_KEY
+from ..utils.http_client import LaunchableClient
+from ..utils.token import parse_token
+from ..utils.java import get_java_command
+
+
+@click.command(name="verify")
+def verify():
+    try:
+        if os.getenv("LAUNCHABLE_TOKEN") is None:
+            click.echo(click.style(f"Could not find the LAUNCHABLE_TOKEN environment variable. Please check if you "
+                                   f"set it properly.", fg="red"))
+            return
+
+        token, org, workspace = parse_token()
+
+        headers = {
+            "Content-Type": "application/json",
+        }
+
+        path = "/intake/organizations/{}/workspaces/{}/verification".format(org, workspace)
+
+        client = LaunchableClient(token)
+        res = client.request("get", path, headers=headers)
+
+        if res.status_code == 401:
+            click.echo(click.style(f"Authentication failed. Most likely the value for the LAUNCHABLE_TOKEN "
+                                   f"environment variable is invalid.", fg="red"))
+            return
+
+        res.raise_for_status()
+
+        click.echo("Platform: " + platform.platform())
+        click.echo("Python version: " + platform.python_version())
+
+        java = get_java_command()
+
+        if java is None:
+            click.echo(click.style(f"Java is not installed. You need Java to use the Launchable CLI", fg="red"))
```

Comment:
Do we need to check Java installation strictly or is it ok a user does not install Java since we have a docker option?

Reply:
I think we need to check Java because the docker option is no longer maintained. The docker option should be deprecated.